arcadia_pycolor/gradient.py

```diff
@@ -39,9 +57,24 @@ def __init__(self, name: str, colors: list[Any], values: Union[list[Any], None]
                 raise ValueError("The first value must be 0 and the last value must be 1.")
             if len(colors) != len(values):
                 raise ValueError("The number of colors and values must be the same.")
-            self.values = values
+            anchor_values = values
         else:
-            self.values = distribute_values(len(self.colors))
+            anchor_values = distribute_values(len(colors))
+
+        self.anchors = [Anchor(color, value) for color, value in zip(colors, anchor_values)]
```

Comment:
surprised the linter doesn't catch this
```suggestion
        self.anchors = [Anchor(color, value) for color, value in zip(colors, anchor_values), strict=True)]
```

Reply:
Edit: oops, nevermind. `strict` was added in 3.10 🤦‍♂️